match_tpl.h

```diff
@@ -55,7 +66,7 @@ Z_INTERNAL uint32_t LONGEST_MATCH(deflate_state *const s, Pos cur_match) {
     /* The code is optimized for MAX_MATCH-2 multiple of 16. */
     Assert(MAX_MATCH == 258, "Code too clever");
 
-    best_len = s->prev_length ? s->prev_length : 1;
+    best_len = s->prev_length ? s->prev_length : MIN_MATCH-1;
```

Comment:
Can use `MAX`?

Reply:
That line is basically conditional move if non-zero, which is not same as `MAX`.